src/components/attendee/AttendeeHeader/AttendeeHeader.module.scss

```diff
@@ -0,0 +1,15 @@
+@use "scss/attendee/variables.scss"as *;
+
+.attendeeHeader {
+  display: flex;
+  position: fixed;
+  z-index: 111;
+  width: 100vw;
+  padding: $whitespace;
```

Comment:
Commented further down to remove the `variables.scss` file and how to replace this and similar ones.

Reply:
[2ebc1f3](https://github.com/sparkletown/sparkle/pull/2827/commits/2ebc1f31eb8b1121e844036f240cd927eeb6d714)